Samples/src/StateVisualizer/package-lock.json

```diff
@@ -0,0 +1,5366 @@
+{
+  "name": "vis-sim",
```

Comment:
```suggestion
  "name": "qs-state-visualizer",
```

Reply:
that's what it generated on my laptop... again, I don't think this should be committed :)

Reply:
Oops, I forgot to update package-lock.json after renaming the package. If you want to add it to gitignore I am also OK with that. :)